jenkinsfiles/ginkgo-kernel.Jenkinsfile

```diff
@@ -12,6 +12,9 @@ pipeline {
         TESTED_SUITE="k8s-${K8S_VERSION}"
         GINKGO_TIMEOUT="300m"
         DEFAULT_KERNEL="419"
+        KERNEL = get_kernel("${DEFAULT_KERNEL}")
```

Comment:
If you set `KERNEL` here in a manner similar to https://github.com/cilium/cilium/blob/master/jenkinsfiles/ginkgo-gke.Jenkinsfile#L128 , it should be available as an env var in all children pipeline steps.